workspaces/marketplace/plugins/marketplace-common/src/api/MarketplaceApi.ts

```diff
@@ -137,4 +137,9 @@ export interface MarketplaceApi {
     namespace: string,
     name: string,
   ): Promise<MarketplacePackage[]>;
+
+  getPackagePlugins(
```

Comment:
Sorry, the current name is a bit unclear. Would you consider using something more precise, such as `findPluginsUsingPackage` or `getPluginsDependingOnPackage`?